lib/src/query_component.dart

```diff
@@ -16,17 +17,20 @@ import 'util.dart';
   template: '''
     <input [(ngModel)]="advertiserId" 
            placeholder="Advertiser ID: 164337" debugId="advertiser-id-input">
+    <input [(ngModel)]="mediaPlanId" 
```

Comment:
can you add a screenshot so to visualize the template changes?

Reply:
Of course, please see [here](https://github.com/googleinterns/dv360-excel-plugin/pull/65#issue-454717402).